packages/libaudit/libaudit.spec

```
@@ -27,6 +27,14 @@ Requires: %{name}
 %description -n %{_cross_os}audit
 %{summary}.
 
+%package -n %{_cross_os}audit-rules
+Summary: Default audit rules for Bottlerocket
+Requires: %{_cross_os}audit
+Requires: %{_cross_os}systemd(journald-audit)
```

It'd be cleaner to have just one of these virtual provides rather than both:

```suggestion
Requires: %{_cross_os}auditd
```

---

The `%{_cross_os}audit` is the package that vends the `auditctl` binary which is required. We can shift that to `%{_cross_os}audit(binaries)` but its just a single binary at this point. So we would need to keep the dependency on the audit, but with shifting to `Requires: %{_cross_os}auditd` it does read a little odd for audit and auditd. 

So I think the new requires would be the below if we wanted to use the Provided from the systemd package. 
```suggestion
Requires: %{_cross_os}audit
Requires: %{_cross_os}auditd
```